packages/xcm-cfg/src/configs/polkadot/pendulum.ts

```diff
@@ -23,6 +23,43 @@ const toHydration: AssetRoute[] = [
     },
     extrinsic: ExtrinsicBuilder().xTokens().transfer(),
   }),
+  new AssetRoute({
+    source: {
+      asset: eurc,
+      balance: BalanceBuilder().substrate().system().account(),
```

Comment:
Are you sure that this is correct? Only the native asset balance can be accessed in the `system::account()` query so I guess this is wrong?

Reply:
Haha yeah I guess it's important, otherwise the balance won't show correctly.